### DISCUSSION TITLE
v1.4.2

### DISCUSSION THREAD

**Discussion by MrMeowzz** [MrMeowzz/Funkin-MrMeowzz#29] [Updates]
- added option in debug menu to change which player notes hitsounds play on
- added option in debug menu to mute voices
- fixed fire note hit sounds not being aligned right
- fixed hitboxes on fire and halo notes

<hr /><em>This discussion was created from the release <a href='https://github.com/MrMeowzz/Funkin-MrMeowzz/releases/tag/v1.4.2'>v1.4.2</a>.</em>

**Comment by DeceptedPar** [comment]
now this is what I call a good engine!

**Comment by DeceptedPar** [comment]
https://mega.nz/folder/YW4SxY5B#t0-d0nNfP256KUa5HX45LQ
might wanna check this out, I made a hard+ ugh using the arrow types
it also has a swapped version
